CMakeLists.txt

```diff
@@ -288,6 +288,11 @@ if(NOT WIN32 AND BUILD_TOXAV AND SNDFILE_FOUND AND PORTAUDIO_FOUND AND OPENCV_FO
     ${OPENCV_LIBRARIES}
     ${PORTAUDIO_LIBRARIES}
     ${SNDFILE_LIBRARIES})
+  # Due to https://github.com/opencv/opencv/issues/6585, we need to compile
+  # av_test as C++ for newer OpenCV versions.
+  if(OPENCV_VERSION VERSION_GREATER 3)
```

Comment:
I think 3.1 is greater than 3. I don't know whether 3 works, but we got the bug report for 3.1. I'll make it "not less than", which is greater equal.